pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -1,48 +1,110 @@
 @import 'color.scss', 'font.scss', 'imports/_breakpoints.scss';
 
 // Code.org HoC homepage banner
-.hoc-banner.homepage {
-  text-align: center;
-  color: $darkest_gray;
-  padding: 7em 0;
-  line-height: 1.4;
+#bars {
+  height: 450px;
+  position: relative;
 
-  @media (min-width: $width-xl) and (max-width: $width-xxl)  {
-    padding: 9em 0;
+  @media screen and (max-width: $width-md) {
+    height: 400px;
   }
 
-  @media screen and (max-width: $width-lg)  {
-    padding: 7em 0 6em;
-  }
+  & > div.col-33 {
+    position: absolute;
```

Comment:
I used absolute positioning here to allow the middle section to be wider than 33% and overlap the left and right images so they could maintain their size w/o overwriting anything in [homepage_hero.haml](https://github.com/code-dot-org/code-dot-org/blob/848673c33968f431efe4d9c1d13ebf583db0bed9/pegasus/sites.v3/code.org/views/homepage_hero.haml)